@xen-orchestra/backups/writers/_MixinBackupWriter.js

```diff
@@ -71,6 +71,5 @@ exports.MixinBackupWriter = (BaseClass = Object) =>
         const remotePath = handler._getRealPath()
         await MergeWorker.run(remotePath)
       }
-      await this._adapter.invalidateVmBackupListCache(this._backup.vm.uuid)
```

Comment:
Attempt to read and inject new backup in cache.

Reply:
Forget this, too complicated (old backups are deleted as well due to retention).